app/controllers/users/edit_email_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Users
+  class EditEmailController < ApplicationController
+    before_action :confirm_two_factor_authenticated
+
+    def edit
+      @update_user_email_form = UpdateUserEmailForm.new(current_user)
+    end
+
+    def update
+      @update_user_email_form = UpdateUserEmailForm.new(current_user)
```

Comment:
I prefer to be explicit, so that if I need to find a particular form in my text editor, I can find the specific form as opposed to getting multiple results for `@form`.